toolkit/src/main/cpp/export/meta_toolkit_export_plugin.cpp

```diff
@@ -176,3 +222,628 @@ PackedStringArray MetaToolkitExportPlugin::_get_android_libraries(const Ref<godo
 
 	return dependencies;
 }
+
+PackedStringArray MetaToolkitExportPlugin::_get_supported_devices() const {
+	PackedStringArray supported_devices;
+
+	if (_get_bool_option("meta_xr_features/quest_1_support")) {
+		supported_devices.append("quest");
+	}
+
+	if (_get_bool_option("meta_xr_features/quest_2_support")) {
+		supported_devices.append("quest2");
+	}
+
+	if (_get_bool_option("meta_xr_features/quest_3_support")) {
+		supported_devices.append("quest3");
```

Comment:
Let's also add `quest3s` as done in https://github.com/GodotVR/godot_openxr_vendors/blob/3b4fb43d4e88be334f0ce73244490216861f46fd/plugin/src/main/cpp/export/meta_export_plugin.cpp#L220